cea/demand/ventilation_model.py

```diff
@@ -565,7 +565,7 @@ def allocate_default_ventilation_openings(coeff_vent_zone, height_zone):
     # initialize
     coeff_vent_path = np.zeros(4)
     height_vent_path = np.zeros(4)
-    orientation_vent_path = np.zeros(4)
+    orientation_vent_path = [0, 1, 0, 1]
```

Comment:
Again. `orientation_vent_path`  is used to retrieve an array index and should be integral. We could also just use `map(int, np.zeros(4))`. But, why not just assign the correct indexes with a list literal? As mentioned above, the code structure still needs some work...